.github/workflows/deployment.yml

```diff
@@ -3,14 +3,14 @@ name: Deployment of Zowe Explorer and related extensions
 on:
   push:
     branches:
-      - master
+      - v1.16.x
```

Comment:
@zFernand0 this will trigger publishing. right?

Reply:
Not for the branch this PR is being merged into (`next`)